playwright-e2e/pages/upload-draft-orders.ts

```diff
@@ -61,6 +61,6 @@ export class UploadDraftOrders extends BasePage {
         await expect(this.page.getByText('draftOrder.docx')).toBeVisible();
 
         await this.clickContinue();
-        await this.checkYourAnsAndSubmit('Submit');
+        await this.checkYourAnsAndSubmit();
```

Comment:
it will click the Save and Continue button now with the changes made in base page and not the Submit button which is expected here I think?

Reply:
i'm not sure why the base page method was changed and not a new one added for submit? as this one is used to click the 'Save and continue' button?